instruqt-tracks/nomad-integration-with-vault/track.yml

```diff
@@ -0,0 +1,518 @@
+    type: code
+    hostname: hashistack-server
+    path: /root/hashistack/
+  - title: Server
+    type: terminal
+    hostname: hashistack-server
+  - title: Nomad
+    type: service
+    hostname: hashistack-server
+    port: 4646
+  - title: Consul
+    type: service
+    hostname: hashistack-server
+    port: 8500
+  - title: Vault
+    type: service
+    hostname: hashistack-server
+    port: 8200
+  difficulty: basic
+  timelimit: 600
+- slug: deploy-an-application
+  id: 4ddvvt3xlnzr
+  type: challenge
+  title: Deploy an Application
+  teaser: Deploy an Application with the Appropriate Policy and Templating
+  assignment: |-
+    The Nomad job file has been also been created for you and you can take a look at it by clicking on the <b>"Files"</b> tab and navigating to the <b>/root/hashistack/nomad/web.nomad</b> file.
+
+    Run the Nomad job by running the command:
+    ```
+    nomad run /root/hashistack/nomad/web.nomad
+    ```
+    Click on the <b>"Nomad"</b> tab and wait until the job has been deployed and healthy.
+
+    Confirm that the application is accessing the database by running two commands. Use the dig command to query the SRV record of your service and obtain the port it is using:
+    ```
+    dig +short SRV web.service.consul.
+    ```
+    The response should look similar to
+    ```
+    1 1 3000 hashistack-client-1.node.instruqt.consul.
+    ```
+    Next curl your service at the appropriate port and names path:
+    ```
+    curl http://web.service.consul:3000/api | jq
+    ```
+    The response should return json from the api.
+
+    Finally, click of the <b>"Application"</b> tab to view the web application. All of the content, links and image locations come from the database.
+
+    Click the <b>"Check"</b> button in the lower right to complete the challenge.
+  notes:
+  - type: text
+    contents: |-
+      <b>Deploy an Application</b>
+      <hr />
+      Now we are ready to deploy our web application and give it the necessary policy and configuration to communicate with our database.
+  - type: text
+    contents: |-
+      There are a few key points to take a look at in the <b>/root/hashistack/nomad/web.nomad</b> file;
+
+      We have specified the access-tables policy in the vault stanza of this job. The <a target="_blank" href="https://nomadproject.io/docs/">Nomad</a> client will receive a token with this policy attached. Recall from the previous step that this policy will allow our application to read from the database/creds/accessdb endpoint in <a target="_blank" href="https://www.vaultproject.io/docs/">Vault</a> and retrieve credentials.
+
+      <pre style="overflow-x: auto; white-space: pre-wrap; white-space: -moz-pre-wrap; white-space: -pre-wrap; white-space: -o-pre-wrap; word-wrap: break-word; padding: 15px;">
+      vault {
+        policies = ["access-tables"]
+      }
+      </pre>
+  - type: text
+    contents: We are using the template stanza's vault integration to populate environment
```

Comment:
You need to fix the formatting here by adding `|-` after `contents:` and then get rid of the line brekas.

Also, you've given the same link to consul template twice.